web-ui/src/components/skeleton_loader/SkeletonLoader.jsx

```diff
@@ -14,51 +16,144 @@ const StyledCard = styled(Card)(() => ({
   }
 }));
 
-export default function SkeletonLoader({type}) {
-    // guild and team currently have the same return value but were given different conditionals
-    // for clarity / in case one changes
-    if (type === "team"){
-        return (
-        <StyledCard className={classes.card}>
-          <CardHeader
-              title={<Skeleton height={50} variant="text" />}
-              subheader={<Skeleton variant="text" />}
-          />
-          <Box mt={2} ml={2} mr={2} height={80}>
-            <Skeleton variant="text" height={15}/>
-            <Box height={10}/>
-            <Skeleton variant="text" height={15}/>
-            <Skeleton variant="text" width={180} height={15}/>
+
+
+const useStyles = makeStyles({
+  noTopBottomPadding: {
+    paddingBottom: 0,
+    paddingTop: 0,
+  },
+  smallMargin: {
+    marginLeft: "1em",
+  },
+  flexShrink: {
+    flexShrink: 1,
+  }
+
+})
+
+export default function SkeletonLoader({ type }) {
+  const additionalClasses = useStyles()
+  // guild and team currently have the same return value but were given different conditionals
+  // for clarity / in case one changes
+  if (type === "team") {
+    return (
+      <StyledCard className={classes.card}>
+        <CardHeader
+          title={<Skeleton height={50} variant="text" />}
+          subheader={<Skeleton variant="text" />}
+        />
+        <Box mt={2} ml={2} mr={2} height={80}>
+          <Skeleton variant="text" height={15} />
+          <Box height={10} />
+          <Skeleton variant="text" height={15} />
+          <Skeleton variant="text" width={180} height={15} />
+        </Box>
+      </StyledCard>
+    );
+  }
```

Comment:
Is Prettier being used to format code?

Reply:
I haven't used it, but I can go back and use Prettier on the code I wrote if you wish!

Reply:
Nearly all OCI web apps use Prettier to format JavaScript and TypeScript code, so I think it's a good idea.

Reply:
I used Prettier on the skeleton loader file in question, and will use it from now on before pushing anything. Thank you for the advice!